sdk/core/azure-core/CMakeLists.txt

```
@@ -120,6 +120,9 @@ target_include_directories(azure-core PUBLIC ${CURL_INCLUDE_DIRS})
 target_include_directories(azure-core INTERFACE ${nlohmann_json_INCLUDE_DIRS})
 target_link_libraries(azure-core INTERFACE CURL::libcurl Threads::Threads nlohmann_json::nlohmann_json)
```

In that case, remove the `CURL::libcurl` INTERFACE from this line:
```suggestion
target_link_libraries(azure-core INTERFACE Threads::Threads nlohmann_json::nlohmann_json)
```